Gemfile

```diff
@@ -42,19 +47,27 @@ group :development, :test do
   # fcd1, 04/11/22: Rails 6 doesn't like '~> 1.3.13'. As ldpd-amesa does, spec to ~> 1.4
```

Comment:
Okay to remove this comment since you're on Rails 7 now?

Reply:
Comment removed.